webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -914,18 +914,16 @@ private static void addFmcScrambleCutoutSheet(PdfWriter docWriter, Document doc,
         final int scramblesPerSheet = 8;
         for (int y = 0; y < scramblesPerSheet; y++) {
             cb.beginText();
-            String title = "";
+            String title = globalTitle + " - " + scrambleRequest.title;
             if(scrambleRequest.scrambles.length > 1) {
-                title = globalTitle + " - " + scrambleRequest.title + " - Scramble " + (index + 1) + " of " + scrambleRequest.scrambles.length + ":";
-            } else {
-                title = globalTitle + " - " + scrambleRequest.title + ":";
+                title += " - Scramble " + (index + 1) + " of " + scrambleRequest.scrambles.length;
             }
 
             fontSize = fitText(new Font(bf), title, new Rectangle(availableScrambleSpace, 100), scrambleFontSize, false, 1f);
 
             cb.setFontAndSize(bf, fontSize);
             cb.showTextAligned(PdfContentByte.ALIGN_LEFT, title, left, top - offsetTop, 0);
-            cb.showTextAligned(PdfContentByte.ALIGN_LEFT, globalTitle + " - " + scrambleRequest.title + ":", left, top - offsetTop, 0);
+            cb.showTextAligned(PdfContentByte.ALIGN_LEFT, globalTitle + " - " + scrambleRequest.title, left, top - offsetTop, 0);
```

Comment:
I'm having trouble understanding how this line of code interacts with the previous line of code. It looks like they're drawing to the exact same coordinates... why aren't they overlapping?

Reply:
These bugs are currently been used on TNoodle. I realized it on my [last competition](https://www.worldcubeassociation.org/competitions/FMCBrasil2019).

I'll replace these line with fixAndShowText. This sould save a few lines and make it more clear.

Reply:
You were correct. There was 2 titles shown at the same spot.
[This](https://github.com/thewca/tnoodle/blob/c798b44b78757bf39516516b32d06c837d256661/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L927) and [this](https://github.com/thewca/tnoodle/blob/c798b44b78757bf39516516b32d06c837d256661/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L928).